candidate-standard/clause_8_collections.adoc

```diff
@@ -41,7 +41,7 @@ The OGC API-Common standard does not define any Information Resource types. Howe
 
 Query parameters are used in URLs to define the resources which are returned on a GET request. The EDR standard defines the following as standard parameters for use in EDR API standards.
 
-==== Parameter coords
+==== Parameter coordinates
```

Comment:
The query parameter is coords not coordinates

Reply:
@m-burgoyne OK got it - I thought I was editting a bit of English in a title.